src/driver/communication_interface.h

```diff
@@ -99,6 +100,9 @@ class CommunicationInterface {
     sim_control_exception_triggered_ = false;
   }
 
+  bool CompliantPushFwdRequested() const { return compliant_push_requested_; }
+  void ClearCompliantPushFwdRequest() { compliant_push_requested_ = false; }
```

Comment:
Can we standardize on just `CompliantPush` everywhere instead of `CompliantPushFwd`? A push backwards is a pull anyhow... 🤔